labshare/models.py

```diff
@@ -123,11 +127,55 @@ class Reservation(models.Model):
     gpu = models.ForeignKey(GPU, related_name="reservations", on_delete=models.CASCADE)
     user = models.ForeignKey(User, related_name="reservations", on_delete=models.CASCADE)
     time_reserved = models.DateTimeField(auto_now_add=True)
+    usage_started = models.DateTimeField(null=True, blank=True)
```

Comment:
I think we should remove `blank=True` since `''` should not be allowed as value of a date

Reply:
Actually this seems about forms, DateTimeField can not be blank anyway. It is stored as NULL instead. https://stackoverflow.com/a/8609425/11367615

Reply:
this is so fucked in Django...